tuna/corrupt_configs.py

```diff
@@ -85,7 +106,7 @@ def main():
         outfile.write(bash_cmd + '\n')
         count += 1
 
-  LOGGER.warning(f"Added {count} entries")
+  LOGGER.warning("Added {count} entries")
```

Comment:
The f before the string here is a formatting queue, it would replace {count} with a value. This should be re-added.

Reply:
LOGGER.warning("Entries Added = %d", count) , this address the W1203 - pylint error